apps/client/src/components/StepByStepNavigation/StepByStepNavigationStyle.ts

```diff
@@ -1,8 +1,10 @@
-import styled from "styled-components";
+import { themeColor } from "@datapunt/asc-ui";
+import styled, { css } from "styled-components";
 
 import StepByStepItemStyle, { CircleWrapperStyle } from "./StepByStepItemStyle";
 
 export type Props = {
+  addBorderBetweenItems?: boolean;
```

Comment:
I know the styling is a border, but i think a 'inbetweenLine' or something is more declarative